File: SomniCorp - Copie/help.py
import discord
from discord.ext import commands

def register_help_commands(bot):
    @bot.command(name="aide")
    async def help_command(ctx):
        embed = discord.Embed(
            title="📖 Manuel d'opérations SomniCorp",
            description="Liste des procédures approuvées par SomniCorp. À utiliser avec précaution.",
            color=discord.Color.teal()
        )
        embed.add_field(name="🧰 Inventaire", value="`/inv` — Voir ton inventaire ou celui d’un membre.", inline=False)
        embed.add_field(name="⚔️ Combat", value="`/fight @membre` — Attaque un joueur.\n`/heal [@membre]` — Soigne.", inline=False)
        embed.add_field(name="🎁 Récompense quotidienne", value="`/daily` — Récupère ta récompense du jour.", inline=False)
        embed.add_field(name="🏆 Classement", value="`/leaderboard` — Affiche le classement.", inline=False)
        embed.set_footer(text="Utilisez /help pour la version slash.")
        await ctx.send(embed=embed)

    @bot.tree.command(name="help", description="Affiche la liste des commandes disponibles")
    async def help_slash(interaction: discord.Interaction):
        embed = discord.Embed(
            title="📖 Manuel d'opérations SomniCorp",
            description="Liste des procédures approuvées par SomniCorp. À utiliser avec précaution.",
            color=discord.Color.teal()
        )
        embed.add_field(name="🧰 Inventaire", value="`/inv` — Voir ton inventaire ou celui d’un membre.", inline=False)
        embed.add_field(name="⚔️ Combat", value="`/fight @membre`, `/heal [@membre]`", inline=False)
        embed.add_field(name="🎁 Récompense quotidienne", value="`/daily` — Réclame un bonus 1x par jour.", inline=False)
        embed.add_field(name="🏆 Classement", value="`/leaderboard` — Voir les dégâts et soins.", inline=False)
        embed.set_footer(text="Réalisez vos rêves, tout simplement — avec SomniCorp.")
        await interaction.response.send_message(embed=embed, ephemeral=True)

File: GotValis/Restimon_botfix-main/commands/help.js
import { SlashCommandBuilder, EmbedBuilder } from 'discord.js';

export const data = new SlashCommandBuilder()
  .setName('help')
  .setNameLocalizations({ fr: 'aide' })
  .setDescription('Affiche la liste des commandes disponibles');

export async function execute(interaction) {
  const embed = new EmbedBuilder()
    .setColor('#5865F2')
    .setTitle('📚 Manuel crypté — Réseau GotValis')
    .setDescription(
      '**Bienvenue dans GotValis !**\n\n' +
      'Un bot Discord complet avec système de combat, loot boxes, économie, gacha, boutique et bien plus !\n\n' +
      '**Liste des interfaces civiles autorisées:**'
    )
    .addFields(
      {
        name: '📊 Profil & Statistiques',
        value:
          '`/profile`\n' +
          '📊 Affiche ton profil GotValis (PV, PB, tickets, or, perso équipé).\n\n' +
          '`/info [user]`\n' +
          '📈 Affiche tes stats d\'activité (messages, vocal, classement, KDA).\n\n' +
          '`/inventaire` (ou `/inv`)\n' +
          '🎒 Ouvre ton inventaire (objets, quantités, tickets, GoldValis).',
        inline: false,
      },
      {
        name: '💰 Économie & Récompenses',
        value:
          '`/daily`\n' +
          '📅 Récupère ton paquet quotidien (ticket + items + or).\n\n' +
          '`/shop`\n' +
          '🏪 Achète ou vends des objets et personnages.\n' +
          '  • `/shop list` - Voir tous les articles\n' +
          '  • `/shop buy <emoji>` - Acheter un objet\n' +
          '  • `/shop sell <objet>` - Vendre un objet\n' +
          '  • `/shop sell-character` - Vendre un personnage',
        inline: false,
      },
      {
        name: '🎴 Gacha & Personnages',
        value:
          '`/summon` (ou `/invocation`)\n' +
          '🎫 Utilise un ticket pour invoquer un personnage.\n\n' +
          '`/equip`\n' +
          '🎭 Équipe un personnage de ta collection.\n\n' +
          '`/unequip`\n' +
          '❌ Déséquipe ton personnage actuel (cooldown: 1h).',
        inline: false,
      },
      {
        name: '📦 Loot & Items',
        value:
          '`/ouvrir`\n' +
          '📦 Ouvre ton dernier loot box reçu.\n\n' +
          '**Loot automatiques:**\n' +
          '• Loot boxes toutes les 5-15 messages (30s, 5 joueurs max)\n' +
          '• Ravitaillements spéciaux (max 3/jour, 5 min, récompenses variées)',
        inline: false,
      },
      {
        name: '⚔️ Combat & Soins',
        value:
          '`/fight <cible> <objet>`\n' +
          '⚔️ Attaque un joueur avec un objet offensif.\n\n' +
          '`/heal <objet> [cible]`\n' +
          '💊 Soigne un joueur avec un objet de soin.\n\n' +
          '`/use <objet> [cible]`\n' +
          '🛡️ Utilise un objet utilitaire (soin, bouclier).',
        inline: false,
      },
      {
        name: '🏆 Classement',
        value:
          '`/lb_set <10|20>`\n' +
          '🏆 Configure le leaderboard (admin).\n\n' +
          '`/lb_refresh`\n' +
          '🔄 Force une mise à jour du leaderboard (admin).\n\n' +
          '`/lb_stop`\n' +
          '🛑 Supprime le leaderboard (admin).',
        inline: false,
      },
      {
        name: '🎮 Autres Commandes',
        value:
          '`/help`\n' +
          '📚 Ce manuel crypté GotValis.',
        inline: false,
      }
    )
    .setFooter({
      text: 'Modules administratifs non listés.\n💡 Astuce: Envoyez des messages pour déclencher des loot boxes automatiques !'
    })
    .setTimestamp();

  await interaction.reply({ embeds: [embed], ephemeral: true });
}
